src/Microsoft.DotNet.Build.Tasks/PackageFiles/PerfTesting.targets

```diff
@@ -19,8 +19,8 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetOS)'=='Windows_NT'">
     <XunitPerfAnalysisPackageId>Microsoft.DotNet.xunit.performance.analysis</XunitPerfAnalysisPackageId>
-    <XunitPerfAnalysisPath>$(PackagesDir)$(XunitPerfAnalysisPackageId)/$(XunitPerfAnalysisPackageVersion)/tools/xunit.performance.analysis.exe</XunitPerfAnalysisPath>
-    <XunitPerfAnalysisDir>$(PackagesDir)$(XunitPerfAnalysisPackageId)/$(XunitPerfAnalysisPackageVersion)/tools</XunitPerfAnalysisDir>
+    <XunitPerfAnalysisPath>$(PackagesDir)$(XunitPerfAnalysisPackageId)\$(XunitPerfAnalysisPackageVersion)\lib\netstandard1.3\xunit.performance.analysis.exe</XunitPerfAnalysisPath>
```

Comment:
Seems reasonable, but if these files now resolve as libraries for the test, even if they aren't technically netstandard, why do you need this variable?  I'd expect once they're included in runtests.cmd your path would be short and relative to or equal to the execution directory.